vignettes/cookbook_advanced_use_cases.Rmd

```diff
@@ -14,10 +14,520 @@ knitr::opts_chunk$set(
+    year == 2030
+  ) %>% 
+  select(sector, technology, metric, year, technology_share) %>% 
+  mutate(technology_share = round(technology_share, 3))
+
+ei_sector_targets <- sda_targets_portfolio %>% 
+  filter(
+    region == "global",
+    emission_factor_metric == "target_demo",
+    year == 2030
+  ) %>% 
+  select(sector, emission_factor_metric, year, emission_factor_value) %>% 
+  mutate(emission_factor_value = round(emission_factor_value, 3))
+
+available_sectors <- unique(pull(filter(matched, sector %in% c(unique(scenario$sector), unique(co2$sector))), sector))
+
+```
+In this example, we set targets for a bank, based on its loan book composition in `r min(scenario$year, na.rm = TRUE)`. The loan book contains exposures to the following PACTA sectors, for which trajectories are available in the scenarios: `r paste(available_sectors, collapse = ", ")`. The bank wants to set targets for the year 2030. The bank has decided to use the "target_sda" scenario as a benchmark for setting targets based on the market share approach and "target_demo" for setting targets based on the sectoral decarbonization approach. Targets are set based on global scenario benchmarks. The following two tables show how targets are set for sectors based on PACTA.
+
+```{r use_case_two_sector_targets_results, echo = FALSE, warning = FALSE}
+result_table(tech_mix_sector_targets, title = "Sector targets", subtitle = "high-carbon share")
+
+result_table(ei_sector_targets, title = "Sector targets", subtitle = "emission intensity")
+```
+
+The sector targets for 2030 based on this loan book are as follows:
+
+- The target high-carbon share in the automotive sector is: `r sum(filter(tech_mix_sector_targets, sector == "automotive", technology == "ice")$technology_share, na.rm = TRUE) * 100`% - which coincides with the technology level target for light-duty vehicles with internal combustion engines (ice)
+- The target high-carbon share in the power sector is: `r sum(filter(tech_mix_sector_targets, sector == "power", technology %in% c("coalcap", "gascap", "oilcap"))$technology_share, na.rm = TRUE) * 100`%
+- If targets are set by technology in the power sector, the targets would look as follows:
+  - share of coal-fired power plants: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "coalcap")$technology_share, na.rm = TRUE) * 100`%
+  - share of gas-fired power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "gascap")$technology_share, na.rm = TRUE) * 100`%
+  - share of oil-fired power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "oilcap")$technology_share, na.rm = TRUE) * 100`%
+  - share of renewable power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "renewablescap")$technology_share, na.rm = TRUE) * 100`%
+  - share of hydro power: `r sum(filter(tech_mix_sector_targets, sector == "power", technology == "hydrocap")$technology_share, na.rm = TRUE) * 100`%
```

Comment:
More user-friendly to display this as a table

Reply:
@jdhoffa I made a new issue for this, we can beautify in a follow up PR #161